tests/e2e/readme.md

```diff
@@ -0,0 +1,62 @@
+# end-2-end tests for kava
+
```

Comment:
might be good place to list depedency on kvtool subrepo (or adding the sub repo init and updating of the sub-repo as a make target that is a dependency of the `test-e2e` target)